cassovary-core/src/test/scala/com/twitter/cassovary/graph/ArrayBasedDynamicDirectedGraphSpec.scala

```diff
@@ -0,0 +1,76 @@
+package com.twitter.cassovary.graph
+
+import org.scalatest.WordSpec
+import com.twitter.cassovary.graph.StoredGraphDir._
+import org.scalatest.matchers.ShouldMatchers
+
+class ArrayBasedDynamicDirectedGraphSpec extends WordSpec with ShouldMatchers with GraphBehaviours {
```

Comment:
See here for the structure of a test:
http://doc.scalatest.org/2.2.1/#org.scalatest.WordSpec

To obtain nice class spec when running test, it should be sth like 
```
"An ArrayBasedDynamicDirectedGraph" should {
   "add new nodes" in {
   }
  "delete edges" in {
  }
}
```